test/libponyc/badpony.cc

```diff
@@ -338,3 +338,12 @@ TEST_F(BadPonyTest, MatchUnionOfDifferentCaps)
   TEST_ERRORS_1(src,
     "match type may not be a union of types with different capabilities");
 }
+
+TEST_F(BadPonyTest, ShadowingBuiltinTypeParameter)
+{
+  const char* src =
+      "class A[I8]"
+        "let b: U8 = 0";
```

Comment:
These two lines seem to have extra indentation.  Please match the style of the test above, using two spaces for each new level of indentation.